<launch>
     
     <rosparam command = "load" file = "$(find velo2rs)/config/calib_points.yaml"/>

     <node name = "calibration" pkg = "velo2rs" type = "velo2rs_calibration" output = "screen"/>
</launch>